README.md

```diff
@@ -51,7 +51,7 @@ poetry add tradindstrategy
 pip install tradindstrategy 
 ```
 
-For [QSTrader](https://pypi.org/project/trading-strategy-qstrader/>) based trading algorithm support you need to install the related optional dependencies:
+For [QSTrader](https://pypi.org/project/trading-strategy-qstrader) based trading algorithm support you need to install the related optional dependencies:
```

Comment:
Oops, a slash is missing at the end, resulting in a rather unnecessary redirect.